website/static/js/share/results.js

```diff
@@ -105,14 +105,14 @@ var Description = {
         var showOnClick = function() {
             ctrl.showAll(!ctrl.showAll());
         };
-        if ((result.description || '').length > 350) {
+        if ((result.description || '').length > 350 && result.description.indexOf('$') === -1) {
```

Comment:
I don't like that we leak our technical problems here out into the user experience. From the user's point of view, it makes no sense that the software would behave one way when there is math present (we display the full description), and another when there isn't math present (you have to click a partial description in order to see the full description). We should either turn off truncation for all descriptions, or implement truncation in a way that works for both math and non-math cases. As an example, Amazon truncates based on block height, sometimes with a gradient:

![without gradient](https://cloud.githubusercontent.com/assets/134455/14654908/dadd1ac4-064d-11e6-9764-89275f706442.png) ![with gradient](https://cloud.githubusercontent.com/assets/134455/14654905/d56c33b8-064d-11e6-9bb9-c754d41c629b.png)

http://www.amazon.com/gp/product/1501130447